common/containers/Tabs/Contracts/components/Deploy/components/DeployHoc/components/DeployModal.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import translate from 'translations';
+import Modal, { IButton } from 'components/ui/Modal';
+import { BroadcastTransactionStatus } from 'libs/transaction';
+
+export interface Props {
+  chainName: string;
+  nodeName: string;
+  nodeProvider: string;
+  handleBroadcastTx(): void;
+  onClose(): void;
+}
+
+export type TDeployModal = typeof DeployModal;
+
+export const DeployModal = (props: Props) => {
+  const {
+    chainName,
+    nodeName,
+    nodeProvider,
+    handleBroadcastTx,
+    onClose
+  } = props;
+
+  const buttons: IButton[] = [
+    {
+      text: translate('SENDModal_Yes', true) as string,
+      type: 'primary',
+      onClick: handleBroadcastTx
+    },
+    {
+      text: translate('SENDModal_No', true) as string,
```

Comment:
The cancel button is hidden under the confirm button. Need to fix the css for this.

Reply:
This issue will now be fixed when merged into development.